src/modules/launcher/PowerLauncher.UI/ResultList.xaml.cs

```diff
@@ -89,5 +91,74 @@ private void UserControl_Loaded(object sender, RoutedEventArgs e)
         public event PropertyChangedEventHandler PropertyChanged;
 
         private void OnPropertyChanged(string propertyName) => PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+
+        private void SuggestionsList_RightTapped(object sender, Windows.UI.Xaml.Input.RightTappedRoutedEventArgs e)
+        {
+            Debug.WriteLine($"Right Tap was triggered on {e.OriginalSource}");

```

Comment:
[nit] Is this intended?

Reply:
Yes.  It only writes in debug, and we can use it to identify if something being right clicked is stealing focus. Basically it should rarely print, but it if does (like on the scrollbar), then we can just verify that its not a control that will steal focus.

Reply:
Sounds good! Thanks